src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -456,7 +456,11 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
                     onClick={() => {
                       setAudioOptionsOpen(prev => !prev);
                     }}
-                    onBlur={() => setAudioOptionsOpen(false)}
+                    onBlur={event => {
+                      if (!event.currentTarget.contains(event.relatedTarget)) {
```

Comment:
In case of blur event, the `relatedTarget` property is an element that gained focus after it was removed from some other element.